3 - Navigation/Lab - Guard.playground/Pages/1. Exercise - Guard Statements.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+/*:
+ ## Exercise - Guard Statements
+
+ Imagine you want to write a function to calculate the area of a rectangle. However, if you pass a negative number into the function, you don't want it to calculate a negative area. Create a function called `calculateArea` that takes two `Double` parameters, `x` and `y`, and returns an optional `Double`. Write a guard statement at the beginning of the function that verifies each of the parameters is greater than zero and returns `nil` if not. When the guard has succeeded, calculate the area by multiplying `x` and `y` together, then return the area. Call the function once with positive numbers and once with at least one negative number.
+*/
+func calculateArea (length:Double, width:Double) -> Double? {
+    guard length > 0 && width > 0 else {
+        return nil
+    }
+    let area = length * width
+    return area
+}
+
+/*:
+ Create a function called `add` that takes two optional integers as parameters and returns an optional integer. You should use one `guard` statement to unwrap both optional parameters, returning `nil` in the `guard` body if one or both of the parameters doesn't have a value. If both parameters can successfully be unwrapped, return their sum. Call the function once with non-`nil` numbers and once with at least one parameter being `nil`.
+ */
+func add (one:Int? , two:Int?) -> Int? {
+    guard
```

Comment:
This much whitespace is unnecessary and inconsistent from your earlier formatting. How about:
```
func add (one:Int? , two:Int?) -> Int? {
    guard let one = one, let two = two else {
            return nil
    }
    return one + two
}
```